db/schema.rb

```diff
@@ -54,8 +54,6 @@
     t.inet     "last_sign_in_ip"
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
-    t.string   "provider"
```

Comment:
Hmm that's weird, because I pulled the master branch into my local master and then created the drawpieces one with the new updated one...